engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -931,44 +930,8 @@ applyItem (fromCStore, (iid, (itemFull, kit))) = do
 
 -- | Ask for a direction and alter a tile in the specified way, if possible.
```

Comment:
This is no longer "in the specified way", because your removed the generality and moved the only specific case, closing, elsewhere.